src/course-unit/course-sequence/sequence-navigation/SequenceNavigation.jsx

```diff
@@ -0,0 +1,110 @@
+import { Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {
+  injectIntl, intlShape, isRtl, getLocale,
+} from '@edx/frontend-platform/i18n';
+import { Button, useWindowSize, breakpoints } from '@edx/paragon';
+import {
+  ChevronLeft as ChevronLeftIcon,
+  ChevronRight as ChevronRightIcon,
+} from '@edx/paragon/icons';
+
+import { useModel } from '../../../generic/model-store';
+import { RequestStatus } from '../../../data/constants';
+import messages from '../messages';
+import { useSequenceNavigationMetadata } from '../hooks';
+import SequenceNavigationTabs from './SequenceNavigationTabs';
+
+const SequenceNavigation = ({
+  intl,
+  unitId,
+  sequenceId,
+  className,
+}) => {
+  const { sequenceStatus } = useSelector(state => state.courseUnit);
```

Comment:
[question] Is it better to use exist selectors to avoid object destructuring and request all `courseUnit` state?
```suggestion
  const sequenceStatus = useSelector(getSequenceStatus);
```